lighthouse-core/test/config/config-test.js

```diff
@@ -663,8 +663,11 @@ describe('Config', () => {
     });
 
     assert.ok(config.audits.length, 'inherited audits by extension');
-    assert.equal(config.audits.length, origConfig.categories.performance.auditRefs.length + 1);
+    // +1 for `is-on-https`, +1 for `full-page-screenshot`.
+    assert.equal(config.audits.length, origConfig.categories.performance.auditRefs.length + 2);
     assert.equal(config.passes.length, 1, 'filtered out passes');
+    assert.ok(config.audits.filter(a => a.implementation.meta.id === 'is-on-https'));
```

Comment:
`assert.ok([])` does not fail but this is the intent correct?

Reply:
ooooooooooooooooooo shiet. good catch!